### DISCUSSION TITLE
Ltree support?

### DISCUSSION THREAD

**Discussion by trompx** [geldata/gel#4121] [General]
Hello,
I searched in the documentation, github issues and discussions but I saw no mention of the ltree postgres field.
Is it supported? If not do you have an ETA?
I am planning to migrate off prisma as they still don't support it (even in raw queries).

**Comment by elprans** [comment]
`ltree` is an extension type in Postgres, though a first-party one, which creates a bit of a logistical trouble for when you use EdgeDB in front of a remote Postgres cluster, which might not have the extension in that particular build.  That said, we are working on an extension system of our own, so custom types, like `ltree` will be a possibility soon.

**Reply by matheushent** [reply]
I just came with the same question. As you said "like ltree will be a possibility soon.", is today "soon"? When is "soon"?
